src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala

```diff
@@ -101,7 +101,9 @@ class ScalafixInterface private (
 
   def rulesThatWillRun(): Seq[ScalafixRule] =
     try scalafixArguments.rulesThatWillRun().asScala
-    catch { case NonFatal(e) => throw new InvalidArgument(e.getMessage) }
+    catch {
+      case e: ScalafixException => throw new InvalidArgument(e.getMessage)
```

Comment:
leaving the silencing mapping to an invalid argument for https://github.com/scalacenter/scalafix/blob/f41dec8fc14c8c96b84f563ece3bd895304e8287/scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixArguments.java#L193-L195